components/ArticleCard.jsx

```diff
@@ -9,10 +10,12 @@ export default function ArticleCard () {
         getArticles()
         .then((res) => {
             setArticles(res)
-            setIsLoading(false)
+            setIsError(false)
         })
         .catch(() => {
             setIsError(true)
+        })
+        .finally(() => {
             setIsLoading(false)
         })
```

Comment:
This .finally is a bit unnecessary, because if your code gets into a .then or a .catch, then that means the loading is done